projects/aca-content/src/lib/aca-content.routes.ts

```diff
@@ -67,7 +80,16 @@ export const CONTENT_ROUTES: ExtensionRoute[] = [
         children: [
           {
             path: '',
-            loadChildren: () => import('./components/viewer/viewer.module').then((m) => m.AppViewerModule)
+            children: [
```

Comment:
this breaks lazy loading?

Reply:
Libraries should not lazy load other libraries, it is app responsibility (angular do not support libraries with lazy loading other libraries)

Reply:
that's exactly my point, this library should not define the "/login" or other routes, root routes is always an application responsibility, not the library

Reply:
reverted back lazy loading